examples/ssd1306_i2c/ssd1306_i2c.c

```diff
@@ -0,0 +1,70 @@
+/* vim: set expandtab ts=4 sw=4 :
```

Comment:
May be you are right, but i thought this vim configuration may be useful for peoples like me who working on several projects with different indenting.